include/shisen_cpp/utility/capture_setting.hpp

```diff
@@ -21,9 +21,8 @@
 #ifndef SHISEN_CPP__UTILITY__CAPTURE_SETTING_HPP_
 #define SHISEN_CPP__UTILITY__CAPTURE_SETTING_HPP_
 
-#include <shisen_interfaces/msg/capture_setting.hpp>
-
 #include "./emptiable.hpp"
```

Comment:
Don't include like this. Use ```shisen_cpp``` lib which was added from our cmake.

```suggestion
#include "<shisen_cpp/{PATH_TO_LIB}>"
```

Reply:
If we're going to use this style, i think we should also update all files too, like [this](https://github.com/ichiro-its/shisen_cpp/blob/feature/using-grpc/include/shisen_cpp/utility.hpp). Else, we can just left it as is.